src/peft/peft_model.py

```diff
@@ -175,15 +175,16 @@ def save_pretrained(
                 Directory where the adapter model and configuration files will be saved (will be created if it does not
                 exist).
             safe_serialization (`bool`, *optional*):
-                Whether to save the adapter files in safetensors format.
-            selected_adapters (`list(str)`,  *optional*):
+                Whether to save the adapter files in safetensors format, defaults to `True`.
+            selected_adapters (`list` of `str`,  *optional*):
                 A list of adapters to be saved. If `None`, will default to all adapters.
-            save_embedding_layers (`Union[bool, str]`, , *optional*, defaults to `auto`):
+            save_embedding_layers (`Union[bool, str]`, *optional*, defaults to `"auto"`):
                 If `True`, save the embedding layers in addition to adapter weights. If `auto`, checks the common
                 embedding layers `peft.utils.other.EMBEDDING_LAYER_NAMES` in config's `target_modules` when available.
-                Based on it sets the boolean flag. This only works for 🤗 transformers models.
+                Based on it sets, the boolean flag. This only works for 🤗 transformers models.
```

Comment:
This was a little difficult to understand. I think it'd be clearer to combine this with the previous sentence about `auto`

```suggestion
                and automatically sets the boolean flag. This only works for 🤗 transformers models.
```